### ISSUE TITLE
Add header comment to generated IG Publisher files

### ISSUE THREAD

**Issue by markkramerus** [FHIR/sushi#309]
When SUSHI generates IG configuration files, SUSHI should add a comment header indicating that the user should not edit the file, because it is autogenerated and could be overwritten. SUSHI should do this even if it is simply copying from `ig-data` to output directory.  This applies at least to `ig.ini`, `package-list.json`, `package.json`, and `menu.xml`. For example, in `menu.xml`, we should have something like this:

```
<!--
 =================== DO NOT EDIT THIS FILE ====================         
-->
<!--  Generated by SUSHI 0.10.0, any modifications will be overwritten.                                  -->
<!--
 =======================================================         
-->
```

**Comment by cmoesel** [comment]
Fixed by #332.

**Comment by cmoesel** [comment]
NOTE: We were not able to add comments to most JSON files as comments are not supported and the FHIR team indicated that adding a `"//"` property could break tooling.  See #332 for details on what _was_ done.
